components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -355,6 +359,27 @@ def validate_logs_dir(self):
         except ValueError as ex:
             raise ValueError(f"logs_directory is invalid: {ex}")
 
+    def load_execution_container_name(self):
+        if self.execution_container is not None:
+            # Accept configured value in clp-config.yml for debug purposes
+            return
+
+        with open(self.os_release_file_path) as os_release_file:
```

Comment:
Any concerns with adding the library to the package? I feel like it's more future proof since it handles comments, `export`, variable substitutions, etc.

Reply:
No other concern except that is an additional dependency. I agree the solution should be more future proof.
Added in the latest push.